src/devices/src/virtio/block/device.rs

```diff
@@ -788,21 +788,22 @@ pub(crate) mod tests {
 
     #[test]
     fn test_addr_out_of_bounds() {
-        let mut block = default_block(default_engine_type_for_kv());
-        // Default mem size is 0x10000
-        let mem = default_mem();
-        let vq = VirtQueue::new(GuestAddress(0), &mem, 16);
-        set_queue(&mut block, 0, vq.create_queue());
-        block.activate(mem.clone()).unwrap();
-        initialize_virtqueue(&vq);
-        vq.dtable[1].set(0xff00, 0x1000, VIRTQ_DESC_F_NEXT | VIRTQ_DESC_F_WRITE, 2);
+        // Read at valid address, with an overflowing length.
```

Comment:
Not sure if this is needed since there are other tests that check the read operation.

Reply:
There are other tests that check the read operation, but none of them checks this specific case AFAICT. Other partial read tests that focus on a sector that goes past the file length.
I think this is useful to have.

I'll move it to `test_read_write` and maybe also add a write test for this as well.

Reply:
Done